env-defaults

```diff
@@ -198,6 +198,13 @@ MAX_BATCH_INPUTS=1000000000000
 # The upper limit on the combined sizes of all the files in a batch
 MAX_BATCH_SIZE_IN_BYTES=5000000000
 
+# The fraction of back-end get work requests we want to trace with open telemetry
+GET_WORK_SAMPLE_RATIO=0.01
+# The fraction of back-end put work result requests we want to trace with open telemetry
+PUT_WORK_SAMPLE_RATIO=0.1
+# The url for the open telemetry exporter daemon
+OPEN_TELEMETRY_URL=http://localhost:4318/v1/traces
```

Comment:
Will this be running locally for everyone? Should we instead default it to empty and ignore tracing if its empty?

Reply:
Only `npm run start` includes the tracing. `npm run start-dev`, etc., do not include the tracing.